### DISCUSSION TITLE
Master Caution when disconnect A/P

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#5206] [Q&A] [answered]
hello together, 

i do not read every BUG report so ... Did you know if there is any plan to implement triggering MasterCaution when desconnecting the Autopilot? 
Is this maybe a current future project?

thank you for any answer :) 

...and thanks again for this AWESOME project!

**Comment by hotshotp** [comment, marked as answer]
Anything that has a master caution/warning irl and isnt in the mod will be done by the fmgc/fwc implementations.
